### ISSUE TITLE
Strange (reproduced) line break behavior

### ISSUE THREAD

**Issue by BlueDrink9** [alxnbl/onenote-md-exporter#27]
**Describe the bug**  

linebreaks are not kept when the paragraph is followed by a particular set of text

**To Reproduce**

I have provided a onenote section file that contains two pages, with mostly identical content. One works, one doesn't.

**Expected behavior**

Expect the line breaks seen in the passing page.

**Logs**

[files.zip](https://github.com/alxnbl/onenote-md-exporter/files/6242258/files.zip)

**Desktop (please complete the following information):**
 - Windows version : Version	10.0.19042 Build 19042 64 bit
 - Office Version : 15.0.5319.1000 32 bit
 -  [Optional] PanDoc Version : 

```
pandoc.exe 2.11.4
Compiled with pandoc-types 1.22, texmath 0.12.1, skylighting 0.10.2,
citeproc 0.3.0.5, ipynb 0.1.0.1
```

**Additional context**

I've tried manually recreating the page with the same text and it seems to be fine. It must be something invisible in the structure that I'm not seeing.

**Comment by alxnbl** [comment]
Hello @BlueDrink9 and thank you for the bug report.

I have been able to reproduce the bug.  The end of your page for an unkown rease cause PanDoc to add unwanted ">" characters. There was a bug in my post-processing function that correct this unwanted PanDoc behavior that was loosing line-breaks.

I will publish soon a new release including the fix.
[pandoc-output.txt](https://github.com/alxnbl/onenote-md-exporter/files/6321305/pandoc-output.txt)

**Comment by alxnbl** [comment]
Fixed in version 0.5.0
